internal/kubernetes/deployment.go

```diff
@@ -51,6 +51,7 @@ func (d *Deployment) Status() manifest.Status {
 		if strings.EqualFold(string(condition.Type), "progressing") &&
 			strings.EqualFold(condition.Reason, "progressdeadlineexceeded") {
 			s.Failed.State = true
+			s.Failed.Message = condition.Message
```

Comment:
I am aware that in the standard [clouddriver](https://github.com/spinnaker/clouddriver/blob/f34fb388d926387959cad32486f64cf7e33842b7/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesDeploymentHandler.java#L153) They map this error to a constant string.
I could change it so this message matches that.

However, if I change it to that constant string I end up hoping that the k8s message never changes. Eventually, It might change and that would make finding the root for these errors more difficult.

The `Message` field is intended to be a [human-readable message](https://github.com/kubernetes/api/blob/a1613d791b65a52dd5cb0ac47fb08e1fa444184e/apps/v1/types.go#L510) so I think using that message is preferable to replacing it with a constant string.

Lmk what you think

Reply:
I agree this is the preferred method.